spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -118,6 +118,15 @@ def most_relevant_ch(traces):
     return relevant_ch
 
 
+def update_dict(d, u):
```

Comment:
What is the purpose of this? Why can't you use `dict.update`?

Reply:
See #303

Reply:
this does a "deep updating"